substrate-node/pallets/pallet-dao/src/tests.rs

```diff
@@ -873,6 +873,37 @@ fn customize_proposal_duration_out_of_bounds_fails() {
             ),
             Error::<TestRuntime>::InvalidProposalDuration
         );
+
+        assert_noop!(
+            DaoModule::propose(
+                RuntimeOrigin::signed(1),
+                2,
+                Box::new(proposal.clone()),
+                b"some_description".to_vec(),
+                b"some_link".to_vec(),
+                Some(14399)
+            ),
+            Error::<TestRuntime>::InvalidProposalDuration
+        );
+    });
+}
+
+#[test]
+fn motion_proposal_threshold_fails() {
+    new_test_ext().execute_with(|| {
+        let proposal = make_proposal(b"some_remark".to_vec());
+
+        assert_noop!(
+            DaoModule::propose(
+                RuntimeOrigin::signed(1),
+                1,
+                Box::new(proposal.clone()),
+                b"some_description".to_vec(),
+                b"some_link".to_vec(),
+                None
+            ),
+            Error::<TestRuntime>::ThresholdTooLow
+        );
```

Comment:
For this specific one about testing bad threshold it could be:
~~~rust
            DaoModule::propose(
                RuntimeOrigin::signed(_),
                MinThreshold::get() - 1,
                ...
            ),
~~~

Reply:
Done!